src/components/itemcontextmenu.js

```diff
@@ -433,102 +413,48 @@ define(['apphost', 'globalize', 'connectionManager', 'itemHelper', 'appRouter',
                     break;
                     }
                 case 'share':
-                    {
                     navigator.share({
                         title: item.Name,
                         text: item.Overview,
                         url: "https://github.com/jellyfin/jellyfin"
                     });
                     break;
-                    }
                 case 'album':
-                    {
-                        appRouter.showItem(item.AlbumId, item.ServerId);
-                        getResolveFunction(resolve, id)();
-                        break;
-                    }
+                    appRouter.showItem(item.AlbumId, item.ServerId);
+                    getResolveFunction(resolve, id)();
+                    break;
                 case 'artist':
-                    {
-                        appRouter.showItem(item.ArtistItems[0].Id, item.ServerId);
-                        getResolveFunction(resolve, id)();
+                    appRouter.showItem(item.ArtistItems[0].Id, item.ServerId);
+                    getResolveFunction(resolve, id)();
                     break;
-                    }
                 case 'playallfromhere':
-                    {
-                        getResolveFunction(resolve, id)();
+                    getResolveFunction(resolve, id)();
                     break;
-                    }
                 case 'queueallfromhere':
                     {
                     getResolveFunction(resolve, id)();
                     break;
                     }
-                case 'convert':
```

Comment:
I was removing convert because users were reporting the feature as broken. As far as I can tell it uses the sync endpoints, the frontend relies on the sync dialog, and we are probably going to rewrite most of the backend code later.